### PR TITLE
Generate Language List

### PR DESCRIPTION
**Pull request by danieldortega77**
Dynamically generate language dropdown from list of languages; appearance has not changed from #38

### CODE REVIEW

File: capstone/src/main/webapp/script.js [hunk 1]
```diff
@@ -294,6 +303,33 @@ function showAvailableAttributes() {
...
+    radio.name = 'languageRadios';
+    radio.type = 'radio';
+    radio.value = lang;
+  
+    const label = document.createElement('label');
+    label.appendChild(radio);
+    label.innerHTML += LANGUAGES[lang];
+
+    const list = document.createElement('li');
+    // list.className = 'active';
```
Comment (chrismohrii) [inline capstone/src/main/webapp/script.js]: nit: Let's delete this commented-out code

**Review by chrismohrii — APPROVED** [review]
LGTM

File: capstone/src/main/webapp/script.js (hunk 1 above)
Comment (danieldortega77) [reply capstone/src/main/webapp/script.js]: Fixed

**Review by olu144 — APPROVED** [review]

**Comment by olu144** [comment]
LGTM

**Review by dslucas — APPROVED** [review]

**Review by dborkan — APPROVED** [review]
LGTM

**Pull request merged**